### DISCUSSION TITLE
Will row grouping play nicely with drag and drop capability?

### DISCUSSION THREAD

**Discussion by jetamartin** [KevinVandy/material-react-table#786] [Q&A]
If  I created an MRT table using both the "row grouping" and "drag and drop" features and I drag and drop the "Group" row will all of that group's children be moved along with the "Group" row?

If not are there any plans to add that functionality?

Thanks.

**Comment by KevinVandy** [comment]
It should work just fine. Just try it or show what you are trying to do
